packages/react/test/specs/components/Slider/Slider-test.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { ReactWrapper } from 'enzyme'
+
+import { mountWithProvider as mount } from 'test/utils'
+import { isConformant, handlesAccessibility } from 'test/specs/commonTests'
+import Slider from 'src/components/Slider/Slider'
+
+const testValue = '30'
```

Comment:
This value is used two times, it will be more obvious if we will just inline it

Reply:
agreed, done, thanks